cli/templateinit.go

```diff
@@ -36,7 +36,7 @@ func templateInit() *cobra.Command {
 
 			_, _ = fmt.Fprintln(cmd.OutOrStdout(), cliui.Styles.Wrap.Render(
 				"A template defines infrastructure as code to be provisioned "+
-					"for individual developer workspaces. Select an example to get started:\n"))
+					"for individual developer workspaces. Select an example to download:\n"))
```

Comment:
The templates are embedded, not downloaded from an external source. I might be misunderstanding though.

Reply:
They download to your local disk from coderd

Reply:
They actually are embedded in the CLI, not from coderd!

Reply:
Oh yea. Good point. Maybe download is the wrong name, but I see what the confusion might be since it creates a directory.

`Select an example to be copied to the active directory`? Unsure exactly what language

Reply:
I like @Emyrk's suggestion:
```suggestion
					"for individual developer workspaces. Select an example to be copied to the active directory:\n"))
```